oak_marketplace_emotion_recognition/utils.py

```diff
@@ -0,0 +1,235 @@
+        top_graph_progress, top_emotion = self.get_current_top_graph_parameters()
+        top_graph_size = int(size / 4)
+        top_graph_emotion_size = (int(top_graph_size / 3), int(top_graph_size / 3))
+        top_graph = EmotionGraph(size=top_graph_size, emotion=top_emotion, emotion_size=top_graph_emotion_size)
+        top_graph.update_graph(top_graph_progress)
+        top_graph_x = int(size / 2 - top_graph.width / 2)
+        top_graph_y = int(size / 2 - top_graph.height / 2)
+        text = f'MOSTLY {top_graph.emotion.upper()} - {self.get_current_emotion_percent(top_graph.emotion)}%'
+        result_statistic = place_class_names_and_percents(result_statistic,
+                                                          [top_graph_x,
+                                                           int(top_graph_y - 2 * Y_OFFSET / 2)],
+                                                           text)
+        overlay_image_alpha(result_statistic, top_graph.graph, top_graph_x, top_graph_y, top_graph.alpha_mask)
+        start_x, start_y = np.clip(result_statistic.shape[0] - 2 * self.graphs_size, 0, result_statistic.shape[0]) , \
+                               np.clip(result_statistic.shape[1] - self.graphs_size - Y_OFFSET, 0, result_statistic.shape[0])
+        for emotion, ellipse_graph in self.graphs.items():
+            if emotion == top_emotion:
+                continue
+            percent = self.get_current_emotion_percent(emotion)
+            text = f'{emotion.upper()} - {percent}%'
+            result_statistic = place_class_names_and_percents(result_statistic, [start_x, int(result_statistic.shape[1] - Y_OFFSET / 2)],
+                                                        text)
+            alpha_mask = ellipse_graph.alpha_mask
+            overlay_image_alpha(result_statistic, ellipse_graph.graph, start_x, start_y, alpha_mask)
+            start_x = np.clip(start_x - self.graphs_size - 15 * X_OFFSET, 0, result_statistic.shape[0])
+        if is_save:
+            cv2.imwrite('result_statistic.png', result_statistic)
+        return result_statistic
+
+
+class EmotionGraph:
+    def __init__(
+        self, background_color: tuple = (28, 28, 30),
+        progress_color: tuple = (145, 82, 225),
+        size: int = 80,
+        thickness: int = 10,
+        start_angle: int = -220,
+        end_angle: int = 40,
+        emotion: str = 'happy',
+        emotion_size: tuple = (25, 25)
+    ):
+        self.background_color = background_color
+        self.progress_color = progress_color
+        self.size = size
+        self.thickness = thickness
+        self.start_angle = start_angle
+        self.end_angle = end_angle
+        self.emotion_size = emotion_size
+        self.emotion_images = {
+            'neutral': np.array(
+                Image.open(
+                    os.path.join(
+                        os.path.dirname(__file__),
+                        "emotions-pack", "neutral.png",
```

Comment:
Don't hard-code paths. Please, add `emotions-path: "images/emoji"` to __init__ function